Gemfile

```diff
@@ -1,7 +1,5 @@
 source 'https://rubygems.org'
 
-ruby '2.3.0'
```

Comment:
https://github.com/alphagov/smart-answers/pull/2796 will fix this in a better way.